lib/google/serverless/exec.rb

```diff
@@ -415,15 +418,18 @@ def new_rake_task name, args: [], env_args: [],
       #     image to use. (Applies only to the "cloud_build" strategy.)
       # @param strategy [String,nil] The execution strategy to use, or `nil` to
       #     choose a default based on the App Engine environment (flexible or standard) or
-      #     Cloud Run environemnts. Allowed values are `nil`, `"deployment"` (which is the
+      #     Cloud Run environments. Allowed values are `nil`, `"deployment"` (which is the
       #     default for App Engine Standard), and `"cloud_build"` (which is the default for
       #     App Engine Flexible and Cloud Run).
       # @param gcs_log_dir [String,nil] GCS bucket name of the cloud build log
       #     when strategy is "cloud_build". (ex. "gs://BUCKET-NAME/FOLDER-NAME")
+      # @param product [Symbol] The serverless product. If omitted, defaults to the
+      #     value returns by {Google::Serverless::Exec.default_product}.
+      #     Allowed values are `APP_ENGINE` and `CLOUD_RUN`.
```

Comment:
```suggestion
      #     Allowed values are {APP_ENGINE} and {CLOUD_RUN}.
```

This should turn them into links to the constant definitions.